images/postgres-operator/tests/smoke-test.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o errtrace -o pipefail -x
+
+TMPDIR="$(mktemp -d)"
+
+function cleanup() {
+    kubectl logs -n ${NAMESPACE} -l application=spilo 
+    kubectl delete -f ${TMPDIR}/minimal-postgres-manifest.yaml -n ${NAMESPACE}
+    rm -rf "$TMPDIR"
+}
+
+trap cleanup EXIT
+
+sleep 30 
```

Comment:
30 seconds is quite long for a static sleep. Can we add a retry instead? keycloak-fips and jenkins have some examples

Reply:
Fixed it